<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>667</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string>
    @import url('https://cdn.jsdelivr.net/gh/zhiyiYo/PyQt-Fluent-Widgets@main/qfluentwidgets.qss');
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TitleLabel" name="TitleLabel">
     <property name="text">
      <string>Fluent QQ - Minecraft 核心下载与安装</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SmoothScrollArea" name="SmoothScrollArea">
     <property name="styleSheet">
      <string notr="true">background: transparent; border: none</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>691</width>
        <height>606</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="IndeterminateProgressBar" name="IndeterminateProgressBar"/>
       </item>
       <item>
        <widget class="SubtitleLabel" name="SubtitleLabel_2">
         <property name="text">
          <string>下载 Minecraft 版本核心</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CardWidget" name="CardWidget_5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="StrongBodyLabel" name="StrongBodyLabel_5">
            <property name="text">
             <string>核心文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ProgressBar" name="version"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="SubtitleLabel" name="SubtitleLabel">
         <property name="text">
          <string>下载 .minecraft 资源</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CardWidget" name="CardWidget_7">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="StrongBodyLabel" name="StrongBodyLabel_12">
            <property name="text">
             <string>资源库文件 libraries</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ProgressBar" name="libraries"/>
          </item>
          <item>
           <widget class="StrongBodyLabel" name="StrongBodyLabel_13">
            <property name="text">
             <string>63.1 MB</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="CardWidget" name="CardWidget_8">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="StrongBodyLabel" name="StrongBodyLabel_14">
            <property name="text">
             <string>指标文件 indexes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ProgressBar" name="indexes"/>
          </item>
          <item>
           <widget class="StrongBodyLabel" name="StrongBodyLabel_15">
            <property name="text">
             <string>379 KB</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="CardWidget" name="CardWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="StrongBodyLabel" name="StrongBodyLabel_6">
            <property name="text">
             <string>对象文件 objects</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="StrongBodyLabel" name="StrongBodyLabel">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ProgressBar" name="objects1"/>
            </item>
            <item>
             <widget class="StrongBodyLabel" name="StrongBodyLabel_7">
              <property name="text">
               <string>99.9 MB</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="StrongBodyLabel" name="StrongBodyLabel_2">
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ProgressBar" name="objects2"/>
            </item>
            <item>
             <widget class="StrongBodyLabel" name="StrongBodyLabel_8">
              <property name="text">
               <string>85.5 MB</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="StrongBodyLabel" name="StrongBodyLabel_3">
              <property name="text">
               <string>3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ProgressBar" name="objects3"/>
            </item>
            <item>
             <widget class="StrongBodyLabel" name="StrongBodyLabel_9">
              <property name="text">
               <string>92.5 MB</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="StrongBodyLabel" name="StrongBodyLabel_4">
              <property name="text">
               <string>4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ProgressBar" name="objects4"/>
            </item>
            <item>
             <widget class="StrongBodyLabel" name="StrongBodyLabel_10">
              <property name="text">
               <string>42.5 MB</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QOpenGLWidget" name="openGLWidget"/>
       </item>
       <item>
        <widget class="CaptionLabel" name="CaptionLabel">
         <property name="text">
          <string>右侧大小数据为前期估算，可能不是真实数据。可能更小或更大。</string>
         </property>
         <property name="lightColor" stdset="0">
          <color alpha="200">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
         <property name="darkColor" stdset="0">
          <color alpha="200">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SmoothScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ProgressBar</class>
   <extends>QProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>IndeterminateProgressBar</class>
   <extends>QProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
